apps/aecore/src/aec_tx_pool.erl

```diff
@@ -128,9 +128,11 @@ push(Tx) ->
     push(Tx, tx_created).
 
 -spec push(aetx_sign:signed_tx(), event()) -> ok | {error, atom()}.
-push(Tx, Event) when ?PUSH_EVENT(Event) ->
+push(Tx, Event = tx_received) ->
     %% Verify that this is a signed transaction.
-    aec_jobs_queues:run(tx_pool_push, fun() -> push_(Tx, Event) end).
+    aec_jobs_queues:run(tx_pool_push, fun() -> push_(Tx, Event) end);
+push(Tx, Event = tx_created) ->
```

Comment:
I see `push/1` arrives to this clause. I see `aec_tx_pool:push/1` called by `aehttp_dispatch_ext` and `aesc_fsm`. (I did not check `aesc_fsm`.) In `aehttp_dispatch_ext` I see it is in POST `/transactions` - that is external in Swagger schema/config and in [documented as "It is meant to be exposed on the Internet"](https://github.com/aeternity/protocol/blame/022ae65e333e1124153da9e1674e1c8c00a0ca5c/epoch/api/README.md#L20). What prevents malicious Internet user to bypass queue?

Reply:
There is a queue for the HTTP-requests, so the user only bypass the second queue?